packages/portal/package.json

```diff
@@ -25,7 +25,7 @@
     "@angular/platform-browser-dynamic": "^16.0.5",
     "@angular/router": "^16.0.5",
     "apollo-angular": "^4.2.1",
-    "graphql": "^16.6.0",
+    "graphql": "^15.8.0",
```

Comment:
the `graphql` dep in this package is a direct dep of the `apollo-angular` package. We need to make sure downgrading the `graphql` won't introduce breaking changes.

Reply:
I am going to revalidate again.

Reply:
update coming; less time this & last week